### ISSUE TITLE
Add new `plumber.url.callback` support while also supporting legacy `plumber.swagger.url`

### ISSUE THREAD

**Issue by schloerke** [rstudio/rstudio#7702]
### System details

    RStudio Edition : Desktop
    RStudio Version : 1.4.388
    OS Version      : macOS 10.15.6
    R Version       : 4.0.0

### Steps to reproduce the problem

```
# Install the dev version of plumber
remotes::install_github("rstudio/plumber")
library(plumber)

# Disable legacy swagger url support
# This should remove the IDE's set option
options(plumber.swagger.url = NULL)

# Run a plumber API
# The plumber api should display in the selected viewer location.
plumb_api("plumber", "04-mean-sum") %>% pr_run()
```

### Describe the problem in detail

Related: rstudio/plumber#658 (files)

The new version of plumber would like to listen to `plumber.docs.callback` (vs `plumber.swagger.url`). Currently it works for both, but I'd like to eventually stop support for `plumber.swagger.url`.

Setting both options doesn't seem like it should hurt anything. This would allow for both legacy support and moving forward.

### Describe the behavior you expected

By running the code above, I'd expect the plumber url callback to be invoked and to be displayed in the IDE's viewer.

- [x] I have read the guide to submitting good bug reports at https://github.com/rstudio/rstudio/wiki/Writing-Good-Bug-Reports . 
- [x] I have installed the latest version of RStudio and confirmed that the issue still persists.
- [NA] If I am reporting a RStudio crash, I have included a diagnostics report. https://support.rstudio.com/hc/en-us/articles/200321257-Running-a-Diagnostics-Report
- [x] I have done my best to include a minimal, self-contained set of instructions for consistently reproducing the issue.

**Comment by kfeinauer** [comment]
@schloerke Would you be willing to sanity check the fix in the latest dailies to confirm it is working as intended?

**Comment by kevinushey** [comment]
Verified fixed in 1.4.1043.
